### PR TITLE
[BD-6]  Add python 3.8 to travis and change python 3.6 with python 3.5

### PR DESCRIPTION
**Pull request by morenol**
## Description
Add python 3.8 to travis tests

## Reviewers
- [x] @ericfab179
- [ ] @awais786

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @morenol! I've created [OSPR-4459](https://openedx.atlassian.net/browse/OSPR-4459) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

File: .travis.yml [hunk 1]
```diff
@@ -23,7 +24,7 @@ before_install:
 install:
   - "pip install demjson"
   - "pip install -r requirements.txt"
-  - if [ "$TRAVIS_PYTHON_VERSION" = "3.6" ]; then pip install -r requirements3.txt; fi
+  - "pip install -r requirements3.txt"
```
Comment (awais786) [inline .travis.yml]: @morenol is it possible to merge both requirements files ? 
You can check from history why they have added this `if` condition.

Comment (morenol) [reply .travis.yml]: Yes, I think that it is possible, that was added when they started to test in borth python 2 and python 3.

**Review by ericfab179 — APPROVED** [review]

**Review by mraarif — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@morenol 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/configuration&entry.867055334=https://github.com/edx/configuration/pull/5772&entry.1484655318=https://github.com/morenol&entry.752974735=2020-05-04+15:08&entry.1917517419=2020-05-08+15:38&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**